### PR TITLE
Add appointment type to CSV doc

### PR DESCRIPTION
**Pull request by jordondornbos**

### CODE REVIEW

**Comment by jordondornbos** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;IQh370CN&#x2F;5131-2-api-for-appointment-types"> (2) API for Appointment Types</a>

File: csv.rst [hunk 1]
```diff
@@ -212,6 +212,40 @@ import_identifier          This identifier must be completely unique, used if yo
 \* Required
 
 
+Appointment Types
+-----------------
+
+You can import appointment types to be used within Handshake.
+
+\*Params**
```
Comment (Skovy) [inline csv.rst]: @jordondornbos what is this? Shouldn't it be `**Params**`?

File: csv.rst [hunk 2]
```diff
@@ -212,6 +212,40 @@ import_identifier          This identifier must be completely unique, used if yo
...
+
+========================================================== ==================================================================
+Header                                                     Value
+========================================================== ==================================================================
+\*name                                                     The name of the appointment type
+description                                                A description of the appointment type
```
Comment (Skovy) [inline csv.rst]: @jordondornbos is this not required? group the required fields at the top together?

File: csv.rst (hunk 1 above)
Comment (jordondornbos) [inline csv.rst]: @Skovy Just matching the other blocks in this file.

File: csv.rst (hunk 2 above)
Comment (jordondornbos) [inline csv.rst]: @Skovy It's not required. I can group the required fields.

**Comment by Skovy** [comment]
@jordondornbos 🚢  :shipit: ⛵

**Pull request merged**